jetty-websocket/websocket-javax-tests/pom.xml

```diff
@@ -15,6 +15,10 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
```

Comment:
Not here, but below `jetty-slf4j-impl` should have `<scope>test</scope>` like everywhere else.

Reply:
Adjusted top level dependencyManagement for `jetty-slf4j-impl` to be `<scope>test</scope>` instead.